README.md

```diff
@@ -3,6 +3,49 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+### Install and Run
+
+First, make sure Python 3 is installed on your system or follow these
+installation instructions for [Mac] or [Ubuntu].
+
+First, get an access key from [fixer.io](http://fixer.io). You need at least
+the basic plan to get exchange rates from a base currency (such as USD).
+
+Convert `config.sample.json` to `config.json`; fill out your parameters.
+
+It's recommended to use a virtualenv:
+
+```bash
+ python3 -m venv ~/.virtualenvs/tap-fixerio
+ source ~/.virtualenvs/tap-fixerio/bin/activate
+ pip install -U pip setuptools
+ pip install -e '.[dev]'
+```
+
+Set up the `target-csv` virtual environment according to the instructions [here](https://github.com/singer-io/target-csv/blob/master/README.md).
+
+These commands will install `tap-fixerio`  with pip, and then run it:
+
+```bash
+  pip install target-csv
+  tap-fixerio
```

Comment:
Doesn't this need to be that `~/.virtualenvs/tap-fixerio/bin/tap-fixerio …` command I mentioned earlier?